data/avro-schema/src/main/resources/avro/net/corda/data/membership/db/response/MembershipPersistenceResponse.avsc

```diff
@@ -0,0 +1,35 @@
+{
+  "type": "record",
+  "name": "MembershipPersistenceResponse",
+  "namespace": "net.corda.data.membership.db.response",
+  "doc": "Internal response envelope for persistence operations over RPC.",
+  "fields": [
+    {
+      "name": "context",
+      "type": "net.corda.data.membership.db.response.MembershipResponseContext",
+      "doc": "Context for the given request"
+    },
+    {
+      "name": "success",
+      "type": "boolean",
```

Comment:
Maybe instead of boolean, it should be a union of error (with errorMessage) and payload?

Reply:
I agree

Reply:
Will apply that now, thanks.

Reply:
Done